server/routerlicious/packages/services/src/deltaManager.ts

```diff
@@ -26,11 +30,33 @@ export class DeltaManager implements IDeltaService {
 	): Promise<ISequencedDocumentMessage[]> {
 		const baseUrl = `${this.internalAlfredUrl}`;
 		const restWrapper = await this.getBasicRestWrapper(tenantId, documentId, baseUrl);
-		const resultP = restWrapper.get<ISequencedDocumentMessage[]>(
-			`/deltas/${tenantId}/${documentId}`,
-			{ from, to, caller },
-		);
-		return resultP;
+		// if requested size > getDeltasRequestMaxOpsRange, breakdown into chunks of size getDeltasRequestMaxOpsRange
+		if (
+			to != undefined &&
+			from != undefined &&
+			to - from - 1 > this.getDeltasRequestMaxOpsRange
+		) {
+			let getDeltasFrom = from;
+			let getDeltasTo = from + this.getDeltasRequestMaxOpsRange + 1;
+			const chunkResultP: Promise<ISequencedDocumentMessage[]>[] = [];
+			while (getDeltasTo <= to && getDeltasTo - getDeltasFrom - 1 > 0) {
+				chunkResultP.push(
+					restWrapper.get<ISequencedDocumentMessage[]>(
+						`/deltas/${tenantId}/${documentId}`,
+						{ getDeltasFrom, getDeltasTo, caller },
+					),
+				);
+				getDeltasFrom = getDeltasTo - 1;
+				getDeltasTo = Math.min(to, getDeltasFrom + this.getDeltasRequestMaxOpsRange + 1);
+			}
+			return (await Promise.all(chunkResultP)).flat();
```

Comment:
`flat` is neat, however I think the order of ops are important for caller. we need to sort on sn